.github/workflows/chaos.yml

```diff
@@ -23,19 +23,22 @@ jobs:
       - uses: actions/checkout@v2
       - run: echo $HOME
       - run: echo $PATH
-      - run: |
+      - name: Set Npm Config
+        run: |
           npm config set -- '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
           NPM_TOKEN=${NPM_TOKEN} yarn install --frozen-lockfile
       - name: Generate program_ids
         run: ./scripts/programs-keys-gen.sh
-      - run: anchor build
+      - name: Build Contracts
+        run: anchor build
         working-directory: contracts
-      - uses: actions/upload-artifact@master
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@master
         with:
           name: target
           path: contracts/target/deploy
-  build_custom_cl_image:
-    name: Build custom CL image
+  Chaos_Testing_Build_Custom_Chainlink_Image:
```

Comment:
It's a nitpick, but I'd prefer `snake_case` here, to be consistent with the style of the workflow file, and then explicit names for all workflow (top lvl) and jobs.